product_docs/docs/pgd/5/durability/commit-scope-rules.mdx

```diff
@@ -2,49 +2,49 @@
 title: Commit scope rules
 ---
 
-Commit scope rules are at the core of the commit scope mechanism. They define what the commit scope will enforce. 
+Commit scope rules are at the core of the commit scope mechanism. They define what the commit scope enforces. 
 
-Commit scope rules are composed of one or more operations combined with an `AND`. 
+Commit scope rules are composed of one or more operations connected by an `AND`. 
 
```

Comment:
This seemed more idiomatic and clear to me

Reply:
Or maybe just be more explicit? 

Commit scope rules are composed of one or more operations that work in combination. Use an `AND` between rules.

Does that work?

Reply:
Go with that then.